tests/end-to-end/regression/5993/Issue5993Test.php

```diff
@@ -0,0 +1,29 @@
+--TEST--
+PHPUnit process is blocked when there's a lot of output and a test with separate process
+--INI--
+error_reporting=-1
+display_errors=1
+display_startup_errors=1
+memory_limit=-1
+zend.assertions=1
+assert.exception=1
+--SKIPIF--
+<?php
+for ($i = 0; $i < 390; $i++) {
+    \trigger_error("error {$i}");
```

Comment:
Are all the inis needed? Isn't standard long fwrite to stderr/stdout enough?

Reply:
if you want us to assist in fixing this problem, you could try to reduce the reproducer to a very minimum.

we suppose the problem is a underlying php-src bug and having a reproducer small enough to report to the php-src project would be valuable.

Reply:
Is the main or the child blocked?

If the main (reading) process, then output to long stdout/stderr must be enough.

If the child process, then I do not understand the problem.

Reply:
I think 3 processes are involved. Needs further investigation